<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Árbol Genealogico</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>120</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Sexo</string>
    </property>
    <widget class="QRadioButton" name="rb_hombre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hombre</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_mujer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mujer</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txt_name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del miembro familiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_agregar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Preguntas</string>
    </property>
    <widget class="QRadioButton" name="rb_herm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hermanos</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_tios">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tíos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_abue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Abuelos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_preguntar">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Preguntar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_pregunta">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_primos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Primos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_sobrinos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sobrinos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_nietos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nietos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_cunados">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cuñados</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Progenitor</string>
    </property>
    <widget class="QComboBox" name="cmb_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>progenitor de</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_agregar_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
